### PR TITLE
Replace bin/flutter.ps1 with bin/flutter.bat

### LINKED ISSUE #7654: flutter.ps1 script not found when run from the regular command prompt
On windows, with the flutter repo in the `PATH`, typing `flutter` from the regular command prompt will not execute the `flutter.ps1` script:

```
C:\projects\flutter\flutter>flutter
'flutter' is not recognized as an internal or external command,
operable program or batch file.
```

If you instead open windows powershell, running `flutter` does locate the ps1 script on the path.

I'm not sure if we want to support a regular command prompt; if so, perhaps we should add a flutter.bat which can bootstrap into the flutter.ps1 script?

/cc @goderbauer

### PR DESCRIPTION
**Pull request by goderbauer**
fixes https://github.com/flutter/flutter/issues/7654

### CODE REVIEW

File: packages/flutter_tools/lib/src/device.dart [hunk 1]
```diff
@@ -269,8 +269,8 @@ abstract class Device {
 
     // Join columns into lines of text
     return table.map((List<String> row) =>
-        indices.map((int i) => row[i].padRight(widths[i])).join(' • ') +
-        ' • ${row.last}');
+        indices.map((int i) => row[i].padRight(widths[i])).join(' - ') +
+        ' - ${row.last}');
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/device.dart]: why? this seems like a net loss.

**Comment by Hixie** [comment]
Is Windows not UTF-8 capable?

File: packages/flutter_tools/lib/src/device.dart (hunk 1 above)
Comment (goderbauer) [inline packages/flutter_tools/lib/src/device.dart]: Windows's cmd cannot print Unicode characters reliably. Why is this a net loss? It's the same Information, just a different separator character? 

New: 
```shell
  Nexus 6P - 84B5T15A10012703 - android-arm - Android 7.1.1 (API 25)
```

Old:
```shell
Nexus 6P • 84B5T15A10012703 • android-arm • Android 7.1.1 (API 25)
```

Comment (Hixie) [inline packages/flutter_tools/lib/src/device.dart]: Flutter is all-in on using the prettiest and most awesomest Unicode in its messages.

**Comment by goderbauer** [comment]
You can programmatically switch cmd's code page to UTF-8. However there is one big problem with that: There is no good way to switch the code page back to the default. Programs that run after flutter (e.g. git status) will assume the default code page and now produce garbage output in UTF-8 mode. It's a mess.

File: packages/flutter_tools/lib/src/device.dart (hunk 1 above)
Comment (devoncarew) [inline packages/flutter_tools/lib/src/device.dart]: You could use different separator chars based on the OS (pretty on non-windows; functional on windows).

**Comment by Hixie** [comment]
RSLGTM

**Comment by Hixie** [comment]
After discussion we decided to remove the special-casing for Windows for now, and we'll have a WindowsTerminal one day that filters out Unicode and says supportsColor=false and suchlike.

**Comment by Hixie** [comment]
(The problem is that the framework outputs Unicode all the time. Imagine hooking up an Android device that had logged output and then running `flutter logs`. You want the logs to be readable.)

**Pull request merged**